_adr/88/index.adoc

```diff
@@ -1,48 +1,64 @@
 ---
 num: 88
-title: Authorization for control plane admin APIs
-status: "Draft" # One of Draft, Accepted, Rejected
+title: Authorization for fleet-manager admin APIs
+status: "Draft"
 authors:
-  - "akoserwal"
+  - "ppaszki"
 tags:
-  - "sso" # e.g. kafka, connectors, registry
+  - "sso"
 applies_padrs: # What PADRs does this ADR apply?
 applies_patterns: # What APs does this ADR apply?
 ---
 
-// Top style tips:
-// * Use one sentence per line
-// * No unexpanded acronyms
-// * No undefined jargon
-
-// No need for a title heading, it's added by the template
+:rover-groups: https://rover.redhat.com/groups
 
 ## Context and Problem Statement
-// What is the background against which this decision is being taken?
+The purpose of this ADR is to define a standardized approach for securing admin API in a fleet manager
```

Comment:
please can you add more context about the admin API & roles?

For example- (reword to make them generic)
To list and review information about Kafka instances, the account must have the kas-fleet-manager-admin-read role.

To list Kafka instances or view and update a Kafka instance, the account needs to have the kas-fleet-manager-admin-write role assigned to it.

To list Kafka instances, or view, update and delete a Kafka instance, the account needs to have the kas-fleet-manager-admin-full role assigned to it

Reply:
thanks @akoserwal - I have added the explanation about these roles and their permissions